Dockerfile

```diff
@@ -1,37 +1,61 @@
-FROM debian:7.8
+FROM ubuntu:xenial
 MAINTAINER Mike Babineau michael.babineau@gmail.com
 
 ENV \
     ZK_RELEASE="http://www.apache.org/dist/zookeeper/zookeeper-3.4.6/zookeeper-3.4.6.tar.gz" \
     EXHIBITOR_POM="https://raw.githubusercontent.com/Netflix/exhibitor/d911a16d704bbe790d84bbacc655ef050c1f5806/exhibitor-standalone/src/main/resources/buildscripts/standalone/maven/pom.xml" \
+    JVMTOP_RELEASE="https://github.com/patric-r/jvmtop/releases/download/0.8.0/jvmtop-0.8.0.tar.gz" \
     # Append "+" to ensure the package doesn't get purged
-    BUILD_DEPS="curl maven openjdk-7-jdk+" \
-    DEBIAN_FRONTEND="noninteractive"
+    BUILD_DEPS="oracle-java8-installer oracle-java8-set-default maven" \
```

Comment:
We may need to stick to openjdk if we are pushing up to a public repo @bossjones